iml-settings-populator.service

```diff
@@ -5,5 +5,7 @@ After=postgresql-9.6.service
 
 [Service]
 WorkingDirectory=/usr/share/chroma-manager
-ExecStart=/bin/bash -c "exec /bin/python ./manage.py print-settings > /var/lib/chroma/iml-settings.conf"
+ExecStart=/bin/bash -c "exec /bin/python ./manage.py print-settings > /tmp/temp-settings.conf"
+ExecStart=mv /tmp/temp-settings.conf /var/lib/chroma/iml-settings.conf
+ExecStart=rm /tmp/temp-settings.conf
```

Comment:
This last line shouldn't be needed.  And without an `-f` it will emit an error if the file is missing.

Reply:
oh you're right, it's being moved, no copied. I'll remove.